scanapi/tree/request_node.py

```diff
@@ -21,6 +21,8 @@
 )
 from scanapi.utils import join_urls, session_with_retry, validate_keys
 
+DEFAULT_CONTENT_TYPE = "application/json"
```

Comment:
maybe this is not the best place to put this constant, but i'm out of creativity 🙄